openssl.go

```diff
@@ -404,3 +403,9 @@ func CheckLeaks() {
 func versionAtOrAbove(major, minor, patch uint) bool {
 	return vMajor > major || (vMajor == major && vMinor > minor) || (vMajor == major && vMinor == minor && vPatch >= patch)
 }
+
+func bigUint64(b []byte) uint64 {
```

Comment:
Seems too easy to mix up with BigInt:

```suggestion
func bigEndianUint64(b []byte) uint64 {
```